integrations/spring/opendal-spring/src/main/java/org/apache/opendal/spring/core/OpenDAL.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.opendal.spring.core;
+
+/**
+ * Interface that specified a basic set of Apache OpenDAL, implemented by {@link DefaultOpenDAL}.
+ */
+public interface OpenDAL {
```

Comment:
Make `OpenDAL` an interface doesn't make sense to me.

Reply:
@Xuanwo Thanks for your reply, I think it depends on how we want to expose API. In springboot, we can write code like. I want to know what your ideal is
```java
    @Autowired
    private OpenDAL openDAL;

    @PostMapping("/write")
    public ResponseEntity<Void> write(@RequestParam String path, @RequestBody String content) {
        openDAL.write(path, content);
        return ResponseEntity.ok().build();
    }
```

Reply:
OpenDAL is the name of our project, and it's better not to treat it as an interface, which is meaningless. The interface exposed to users should be meaningful, like `operator` or something similar.

---

Would you like to demonstrate how users will utilize the API? I'm more concerned about its appearance to our users.

Reply:
@Xuanwo I think I had posted an example. Let's omit some of the steps for importing dependencies and configuration files. Then, users can declare the following using Spring's injection functionality:

```java
@Autowired
private OpenDAL openDAL;
```

After that, they can use the methods provided by `openDAL` in this file (of course, it can be named more simply, like `dal`). For example, the `write` method:

```java
@PostMapping("/write")
public ResponseEntity<Void> write(@RequestParam String path, @RequestBody String content) {
    openDAL.write(path, content);
    return ResponseEntity.ok().build();
}
```

From my perspective, `OpenDAL` already has sufficient identification (DAL, implying "let's operate on data"). `OpenDALOperations` is also an option, though it's a bit longer. 

BTW, in the OpenGemini project, I used `Operations` because OpenGemini is a database, which wasn’t descriptive enough by itself. https://github.com/openGemini/opengemini-client-java/pull/96

Reply:
I'm not comfortable with `openDAL.write(path, content);`.

Reply:
@Xuanwo Fixed, PTAL